example/useBLASInAlpaka/src/useBLASInAlpaka.cpp

```diff
@@ -0,0 +1,167 @@
+#include <alpaka/alpaka.hpp>
+
+#include <cublas_v2.h>
+
+#include <cmath>
+#include <iostream>
+
+// Index type
+using Idx = std::size_t;
+// Set data type
+using DataType = float;
+
+// Initialize the matrix in column-major order (1D buffer)
+void initializeMatrix(DataType* buffer, Idx rows, Idx cols, int value)
+{
+    for(Idx j = 0; j < cols; ++j)
+    {
+        for(Idx i = 0; i < rows; ++i)
+        {
+            buffer[i + j * rows] = static_cast<DataType>(value);
+        }
+    }
+}
+
+int main()
+{
+    using Dim1D = alpaka::DimInt<1>;
+
+    // Define matrix dimensions, A is MxK and B is KxN
+    Idx const M = 4; // Rows in A and C
+    Idx const N = 2; // Columns in B and C
+    Idx const K = 3; // Columns in A and rows in B
+
+    // Define device and queue
+    using Acc = alpaka::AccGpuCudaRt<Dim1D, Idx>;
```

Comment:
Could you please use the CUDA tag and derive the ACC from the tag? THis will reduce the work as soon as we refactor the accelerators.

Reply:
I used the standard tags like other examples, but prevented the configuration of this example at cmake if ACC_CUDA_ONLY cmake variable is not set.

Reply:
Anyway i used cuda tag as  you suggested. This example could have a direct main rather than using ExampleTags since only will run with single backend.

`using Acc = alpaka::TagToAcc<alpaka::TagGpuCudaRt, Dim1D, Idx>;`

Reply:
@psychocoderHPC I agree with @mehmetyusufoglu . It does not make sense to use the same template, like in the other examples. The code can be only used with the CUDA backend. Therefore we need no complicated iteration over the enabled tags.